### PR TITLE
[12.0][sale-expense] deactivate sale_order_rule_expense_user

### PR DESCRIPTION
**Pull request by GSLabIt**
Description of the issue/feature this PR addresses: see https://github.com/OCA/OpenUpgrade/issues/2078

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
why?

**Comment by GSLabIt** [comment]
'cause in v 11.0 is True and on v 12.0 has been set to False, see issue for more info.

Ref. : https://github.com/hmrodrigues/odoo/commit/f8e41c6b00131bf387843c24d1003310cde92a5e

**Comment by GSLabIt** [comment]
@mreficent.. any chance to review?

File: addons/sale_expense/migrations/12.0.1.0/noupdate_changes.xml [hunk 1]
```diff
@@ -4,4 +4,7 @@
     <field name="invoice_policy">delivery</field>
     <field name="expense_policy">cost</field>
   </record>
+    <record id="sale_expense.sale_order_rule_expense_user" model="ir.rule">
```
Comment (MiquelRForgeFlow) [inline addons/sale_expense/migrations/12.0.1.0/noupdate_changes.xml]:
```suggestion
    <record id="sale_order_rule_expense_user" model="ir.rule">
```

File: addons/sale_expense/migrations/12.0.1.0/noupdate_changes.xml [hunk 2]
```diff
@@ -4,4 +4,7 @@
...
+    <field name="active">False</field>
```
Comment (MiquelRForgeFlow) [inline addons/sale_expense/migrations/12.0.1.0/noupdate_changes.xml]:
```suggestion
    <field name="active" eval="False"/>
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by GSLabIt** [comment]
@mreficent.. sorry for the question.. first time for me in resolving conversations :) Changes are OK for me.. should I resolve to accept that?

thanks

**Comment by MiquelRForgeFlow** [comment]
Yeah, I would like them to be included.

**Comment by GSLabIt** [comment]
@mreficent, done! thanks

**Pull request merged**